app/src/main/java/org/oppia/android/app/administratorcontrols/appversion/AppVersionViewModel.kt

```diff
@@ -15,7 +15,6 @@ import javax.inject.Inject
 /** [ViewModel] for [AppVersionFragment]*/
 @FragmentScope
 class AppVersionViewModel @Inject constructor(
-  fragment: Fragment,
```

Comment:
You should remove the unused `import` related to this too.

Reply:
@rt4914 i've added description on the PR.
And after removing import and reverting the file how should i push changes?

Reply:
@YashKr01 I think you have already done that.